data/saas/saas_connector_scaffolding/new_fixtures.jinja

```diff
@@ -0,0 +1,113 @@
+from typing import Any, Dict, Generator
```

Comment:
These templates will change in the final version but I kept them here as an example